blog/2025-02-10-pecanrfp.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: pecanrfp
+title: PEcAn Request for Proposals
+authors: david_lebauer
+tags: [funding]
+---
+
+# **Call for Proposals: PEcAn Project Community Development Fund**
+
+The PEcAn Project is excited to announce the availability of funding for small projects and activities. This fund is made possible by GSOC mentor stipends.
+
+We invite proposals for small grants to support activities that advance and promote PEcAn-related work.
+
+## **Supported Activities**
+
+The objective of this funding is to build and support the PEcAn community. A broad scope of activities could be supported. Examples include:
+
+- **Community Engagement:** PEcAn-related swag or outreach activities that enhance community visibility and engagement.
+- **Workshops:** Funding to help coordinate in-person PEcAn-related meetups.
+- **Conference Participation:** Support for presenting PEcAn-related work at conferences.
+- **Publications:** Assistance with publication costs for PEcAn-related papers.
+- **Other Uses:** We are open to creative proposals that support the PEcAn community.
+
+## **Expected Award Size**
+
+All proposals will be considered contingent on available funds. We anticipate typical awards between $500-$3000 USD.
+
+## **Eligibility**
+
+- Open to all PEcAn community members.
+
+## **Out of Scope / Excluded Activities**
+
+- Activities that already have funding.
+- PEcAn Software Development.
+- Travel expenses for PEcAn workshops.
+
+## **How to Apply**
+
+Submit a brief proposal (1-2 pages) that includes:
```

Comment:
How/where should proposals be submitted? Probably the pecan gmail, but the danger is it;s not checked super freuqently

Reply:
Oh, good point! How about a simple google form with fields name, email address, and file upload?

I can also add that those interested should discuss their ideas in the Slack #pecan channel before submission.

Reply:
Google forms are probably even worse, as by default they give the owner no notification at all that something has been submitted, and even if you enable notifications they don't include the submitted form itself. I don't know if they can be set up to send notifications to a whole bunch of people, in which case we're back to deciding who gets notified. While imperfect, emailing pecanproj@gmail.com is probably fine.

Reply:
Added email.